Gemfile

```diff
@@ -3,11 +3,19 @@ source 'https://rubygems.org'
 group :test do
   gem 'rake'
 
-  puppetversion = ENV.key?('PUPPET_VERSION') ? "#{ENV['PUPPET_VERSION']}" : ['>= 3.0.0','< 4.0']
+  puppetversion = if ENV['PUPPET_VERSION']
+                    ENV['PUPPET_VERSION'].to_s
```

Comment:
This logic was blindly taken from another 3rd-party Puppet module...

From the Ruby docs: "Retrieves the value for environment variable name as a String. Returns nil if the named variable does not exist."

Will fix...